client/extensions/woocommerce/app/order/order-details/add-items.js

```diff
@@ -0,0 +1,42 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import OrderFeeDialog from './fee-dialog';
+
+class OrderAddItems extends Component {
+	state = {
+		showDialog: false,
+	};
+
+	toggleDialog = type => () => {
+		this.setState( { showDialog: type } );
+	};
+
+	render() {
+		const { translate } = this.props;
+		return (
+			<div className="order-details__actions">
+				<Button borderless onClick={ this.toggleDialog( 'product' ) }>
+					{ translate( '+ Add Product' ) }
```

Comment:
Optional: What would you think about adding the + using CSS instead, both here and below?

Reply:
+1 on @kellychoffman's idea to use gridicons here.